lab_5_scrapper/scrapper.py

```diff
@@ -120,13 +234,22 @@ def __init__(self, full_url: str, article_id: int, config: Config) -> None:
         """
         Initializes an instance of the HTMLParser class
         """
-        pass
+        self.full_url = full_url
+        self.article_id = article_id
+        self.config = config
```

Comment:
should it be public?

Reply:
if you believe so, could you write down your thoughs as for why?

Reply:
done protected